src/posts/2025-12-31-tetragon-2025-review/index.md

```diff
@@ -0,0 +1,105 @@
+---
+path: '/blog/2025/12/31/tetragon-2025-review'
+date: '2025-12-31T12:00:00.000Z'
+title: 'Tetragon: An End-of-Year 2025 Reflection'
+ogImage: cover.png
+isFeatured: false
+ogSummary: 'Reflections on Tetragon for the year 2025'
+categories:
+  - Community
+tags:
+  - Tetragon
+---
+
+**_Author: Paul Arah, Isovalent@Cisco_**
+
+![](cover.png)
+
+As we close out another incredible year, we want to pause and say thank you to the entire Tetragon community. From contributors and maintainers to users, speakers, and the entire community, this year was shaped by your energy, curiosity, and commitment to advancing the standard of runtime security forward.
+
+Tetragon has continued to grow because of the amazing people behind it: those writing code, testing edge cases, sharing feedback, publishing blogs, and teaching others what’s possible with eBPF-powered security observability and runtime enforcement. Together, we made meaningful progress both technically and as a community, laying a strong foundation for the future.
+
+Below are some of the highlights that defined this year.
+
+## Major Feature Highlights
+
+### Tetragon for Windows (Preview)
+
+This year marked an important milestone with the ongoing work to bring Tetragon’s security observability and runtime enforcement to the Windows ecosystem. The current preview already supports `process_exec_` and `process_exit_` events on Windows, opening the door to consistent, cross-platform visibility. While there’s more work ahead, this is a significant step toward expanding Tetragon’s reach beyond Linux.
+
+### Persistent Enforcement
+
+First introduced in the Tetragon 1.12 release in late 2024, persistent enforcement remains one of the most impactful features that shipped lately. With this persistent enforcement, enforcement policies continue running even if the Tetragon agent goes down, closing a critical security gap during agent restarts or failures. To enable persistent enforcement, configure Tetragon with `--keep-sensors-on-exit`. This ensures that when the tetragon process exits, enforcements policies stays active because they’re pinned in sysfs bpf tree under `/sys/fs/bpf/tetragon` directory.
```

Comment:
```suggestion
First introduced in the Tetragon 1.12 release in late 2024, persistent enforcement remains one of the most impactful features. With persistent enforcement, policies continue running even if the Tetragon agent goes down, closing a critical security gap during agent restarts or failures. To enable persistent enforcement, configure Tetragon with `--keep-sensors-on-exit`. This ensures that when the Tetragon process exits, enforcements policies stays active because they’re pinned in sysfs bpf tree under the `/sys/fs/bpf/tetragon` directory.
```